mit_frontend/src/routers/Login.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import {Card, CardTitle} from 'material-ui/Card';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import '../App.css';
+import styles from './styles';
+
+
+class Login extends Component {
```

Comment:
Where in Login code will call Joseph's or backend code to retrieve user account from DB?

Reply:
It seems like @BruceWeng is working on the UI boilerplate first because the backend isn't ready. So we don't see the REST API call to retrieve user account from the backend yet.